Bio/PDB/Polypeptide.py

```diff
@@ -194,14 +186,45 @@ def is_aa(residue, standard=False):
     >>> is_aa('FME', standard=True)
     False
     """
-    # TODO - What about special cases like XXX, can they appear in PDB files?
     if not isinstance(residue, str):
-        residue = residue.get_resname()
+        residue = f"{residue.get_resname():<3s}"
+    residue = residue.upper()
+    if standard:
+        return residue in protein_letters_3to1
+    else:
+        return residue in protein_letters_3to1_extended
+
+
+def is_na(residue, standard=False):
```

Comment:
Is there a strong reason for this function name? Is seems confusing given the numerical meaning of NA.

Would something like ``is_nuc`` or ``is_nucleotide`` be just as good?

Reply:
No, `is_nucleotide` is indeed better!